ads-api/src/main/java/io/voodoo/apps/ads/api/AdClient.kt

```diff
@@ -284,17 +287,22 @@ abstract class BaseAdClient<ActualType : PublicType, PublicType : Ad>(
         adClickListeners.remove(listener)
     }
 
-    override fun destroyAdsIf(predicate: (Ad) -> Boolean) {
+    /**
+     * @return number of elements being destroyed
+     */
+    override fun destroyAdsIf(predicate: (Ad) -> Boolean) : Int {
         synchronized(loadedAds) {
             val renderedAds = loadedAds.filter(predicate)
```

Comment:
hum maybe rename the "renderedAds" to "adsToDestroy"

Reply:
inded